packages/interceptors-opentelemetry/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@temporalio/interceptors-opentelemetry",
   "version": "1.0.0-rc.0",
   "description": "Temporal.io SDK interceptors bundle for tracing with opentelemetry",
-  "main": "lib/workflow/index.js",
-  "types": "./lib/workflow/index.d.ts",
+  "main": "lib/index.js",
+  "types": "./lib/index.d.ts",
```

Comment:
This might be breaking for people importing workflow code from `@temporalio/interceptors-opentelemetry`?

In our sample, we import from `@temporalio/interceptors-opentelemetry/lib/workflow`: https://github.com/temporalio/samples-typescript/blob/bd2e6d73dcc5de5ff41215a74745ab363ee7e52a/interceptors-opentelemetry/src/workflows.ts#L5

Reply:
It's fine, users should import from the individual subpackages.